ide/atom/README.md

```diff
@@ -0,0 +1,50 @@
+[Packages](#packages) | [Shortcuts](#shortcuts) | [CLI](#cli) | [Feedback](#feedback)
+
+# Atom
+
+Download: <https://atom.io/>
+
+## Packages
+
+- atom-beautify
+- busy-signal
+- editorconfig
+- fonts
+- highlight-selected
+- intentions
+- linter
+- linter-eslint
+- linter-stylelint
+- linter-ui-default
+- minimap
+- pigments
+- pretty-json
+- react2
+- tabs-to-spaces
+
+#### Installation
+
+```
+apm install atom-beautify busy-signal editorconfig fonts highlight-selected intentions linter linter-eslint linter-stylelint linter-ui-default minimap pigments pretty-json react tabs-to-spaces
+```
+
+## Shortcuts
+
+- [Mac](./mac.md)
```

Comment:
Should probably have a single page of shortcuts that displays the shortcut keys for mac and windows.